cmd/fyne/internal/commands/build.go

```diff
@@ -60,6 +66,24 @@ func (b *builder) build() error {
 	return err
 }
 
+func (b *builder) generateMetaLDFlags() string {
+	var vars []string
+	if b.id != "" {
+		vars = append(vars, fmt.Sprintf("-X 'fyne.io/fyne/v2/internal/app.MetaID=%s'", b.id))
+	}
+	if b.name != "" {
+		vars = append(vars, fmt.Sprintf("-X 'fyne.io/fyne/v2/internal/app.MetaName=%s'", b.name))
+	}
+	if b.version != "" {
+		vars = append(vars, fmt.Sprintf("-X 'fyne.io/fyne/v2/internal/aapp.MetaVersion=%s'", b.version))
+	}
+	if b.buildNum != 0 {
+		vars = append(vars, fmt.Sprintf("-X 'fyne.io/fyne/v2/internal/app.MetaBuild=%d'", b.buildNum))
```

Comment:
This is far from performance sensitive code, but isn't Sprintf rather unnecessary here?

Reply:
Would you prefer strconv and a “+”?

Reply:
You can do however you like with that one, I guess. The ones above (which I now released was the intended source of this comment) through, are just strings so they should be as easy as a a simple concatenation.

Reply:
Ah I see, good point